--- colors.default.m4	2003-01-09 12:04:24.000000000 +0000
+++ colors.bright.m4	2003-01-09 12:03:46.000000000 +0000
@@ -1,20 +1,19 @@
-define(FG_DEFAULT,	lightgray)
+define(FG_DEFAULT,	yellow)
 
-define(CL_KEYWORD,			yellow)
-define(CL_SYMBOL,			yellow)
-define(CL_STRING,			green)
-define(CL_COMMENT,			brown)
-define(CL_SEPARATOR,		brightcyan)
-define(CL_NUMBER,			cyan)
+define(CL_KEYWORD,			white			BG_DEFAULT)
+define(CL_SYMBOL,			white			BG_DEFAULT)
+define(CL_STRING,			brightcyan		BG_DEFAULT)
+define(CL_COMMENT,			lightgray		BG_DEFAULT)
+define(CL_SEPARATOR,		CL_KEYWORD)
+define(CL_NUMBER,			brightgreen		BG_DEFAULT)
 
-define(CL_ALT_KEYWORD,		cyan)
+define(CL_ALT_KEYWORD,		brightmagenta	BG_DEFAULT)
